config/config.example.yml

```diff
@@ -314,6 +314,14 @@ https_only: false
 ##
 channel_threads: 1
 
+##
+## Time between two jobs for crawling videos from channels
+##
+## Accepted values: a valid time interval (like 1h30m or 90min)
+## Default: 30m
+##
+channel_refresh_interval: 30min
```

Comment:
```suggestion
#channel_refresh_interval: 30m
```

Reply:
The variable should be commented out because the variable is optional (not as important as `db`, e.g). Also, changing `min` for `m` to stay consistent with the default value in the comment above.